hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt

```diff
@@ -24,13 +24,17 @@ data class HmppsSqsProperties(
     val visibilityTimeout: Int = 30,
     val errorVisibilityTimeout: Int = 0,
     val propagateTracing: Boolean = true,
+    val fifoQueue: String = "false",
+    val fifoThroughputLimit: String? = null,
   )
 
   data class TopicConfig(
     val arn: String = "",
     val accessKeyId: String = "",
     val secretAccessKey: String = "",
     val propagateTracing: Boolean = true,
+    val fifoTopic: String = "false",
+    val contentBasedDeduplication: String = "false",
```

Comment:
Should these be booleans?

Also wondering if you're missin validation in this class, e.g. if contentBasedDuplication is true but fifoTopic is false then should that be an error?